revolv/lib/tests/test_autorepay.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+from cStringIO import StringIO
+
+import mock
+from django.core.management import call_command
+from django.test import TestCase
+from django.utils import timezone
+from revolv.project.models import (MonthlyOrganizationalCost,
+                                   MonthlyReinvestableAmount, Project)
+
+
+class AutoRepayTestCase(TestCase):
+    def test_off_day(self):
+        with mock.patch.object(timezone, 'now', return_value=datetime.datetime(2016, 3, 4)):
+            err_stream_fail = StringIO()
+            try:
+                call_command('autorepay', override=False, stderr=err_stream_fail)
+                self.fail("shouldn't succeed")
+            except SystemExit:
+                pass
+
+            call_command('autorepay', override=True)
+
+    def test_amount(self):
+        project = Project.factories.completed.create(funding_goal=1200)
+        cur_year = timezone.now().year
+        MonthlyReinvestableAmount(
+            project=project,
+            year=cur_year,
+            amount=200
+        ).save()
```

Comment:
You can do `.objects.create(...)` instead of `(...).save()`...I don't really know if that's any better though haha, so up to you